package/MDAnalysis/analysis/pca.py

```diff
@@ -355,12 +356,12 @@ def n_components(self, n):
                 n = len(self._variance)
             self.results.variance = self._variance[:n]
             self.results.cumulated_variance = (np.cumsum(self._variance) /
-                                       np.sum(self._variance))[:n]
+                                               np.sum(self._variance))[:n]
```

Comment:
Don't format code that you didn't touch. We want to keep the diff just referring to your functional changes.

Reply:
Sorry :( I am using Visual Studio Code IDE with autopep8 extension for automatic formatting: that seems to have changed the two lines that I didn't touch. I can manually edit them back , would that be a solution?

Reply:
Generally, yes, that's what I would say... however, looking at this case again and now knowing that it's pep8, let's keep it.